src/views/TradingCompetition/components/YourScore/UserPrizeGrid.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  BlockIcon,
+  CheckmarkCircleIcon,
+  Flex,
+  CrownIcon,
+  Text,
+  TeamPlayerIcon,
+  TrophyGoldIcon,
+} from '@pancakeswap-libs/uikit'
+import useI18n from 'hooks/useI18n'
+import { UserTradingInformationProps } from '../../types'
+import { UseCompetitionCakeRewards, getRewardGroupAchievements } from '../../helpers'
+
+const Td = styled.td`
+  padding: 4px 0;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    padding: 8px;
+  }
+`
+const BoldTd = styled(Td)`
+  font-weight: 600;
+`
+
+const PrizeTable = styled.table`
+  width: 100%;
+
+  thead {
+    border-bottom: 2px solid ${({ theme }) => theme.colors.borderColor};
+  }
+
+  th,
+  td {
+    text-align: center;
+    vertical-align: middle;
+  }
+
+  & > thead th {
+    font-size: 12px;
+    padding: 16px 0;
+    text-transform: uppercase;
+
+    ${({ theme }) => theme.mediaQueries.xs} {
+      padding: 16px 8px;
+    }
+
+    ${({ theme }) => theme.mediaQueries.sm} {
+      padding: 16px;
+    }
+  }
+`
```

Comment:
I think we should not repeat this, instead break the table styles out into its own component.

Reply:
Yea considered this. OK nice one will add 👍